components/benefit_sponsors/app/models/benefit_sponsors/services/invitation_email_service.rb

```diff
@@ -25,6 +26,15 @@ def send_broker_successfully_associated_email
         create_secure_message(message_params, broker_person, :inbox)
       end
 
+      def send_general_agency_successfully_associated_email
+        broker_agency_profile = BenefitSponsors::Organizations::BrokerAgencyProfile.find(get_bson_id(broker_agency_profile_id))
+        general_agency = broker_agency_profile.default_general_agency_profile
+        subject = "You are associated to #{broker_agency_profile.organization.legal_name}- #{general_agency.legal_name} (Hire)"
```

Comment:
this is a hardcoded string, can you re-write it to use the translation files, next line has the same issue

Reply:
Changes added.